docs/config.md

```diff
@@ -141,6 +141,20 @@ ui.relative-timestamps = true
 False by default, but setting to true will change timestamps to be rendered
 as `x days/hours/seconds ago` instead of being rendered as a full timestamp.
 
+### Author format
+
+```toml
+ui.log_author_format = 'username'
```

Comment:
typo: `ui.log-author-format` (dash instead of underscore)

Can you make another PR to fix it?

Reply:
Whoops, sorry! Sent #1236.